src/components/utilities.js

```diff
@@ -22,3 +22,15 @@ export function ResultWrapper(props) {
 
   return field.field === 'id' ? <a href={"https://n2t.net/" + value}>{parse(text)}</a> : parse(text)
 }
+
+
+// functiona to sort the searchFields so the "text" type facets will be in the front
+// and "non-facet" type facets will be in the back
+// keep for right now in case we need to use other sort ways.
+export function sortSearchFields(fields) {
+  const textType = fields.filter((field) => field.type === 'text');
+  const otherTypes = fields.filter((field) => field.type !== 'text' && field.type !== 'non-facet');
+  const nonType = fields.filter((field) => field.type === 'non-facet');
+
+  return [...textType, ...otherTypes, ...nonType];
+}
```

Comment:
This is a helper function to reorder the `searchFields`. We didn't use it yet. Just keep it in case we need to use the new way to order the fields.